_includes/navMenus/build.html

```diff
@@ -5,7 +5,7 @@
             <span class="subline">Build a client app, explore use cases, and build on our demos and resources</span>
             <a href="https://developer.confluent.io/#designbuildrun" class="navItemGroupCta">
                 Start Building
-                <img src={{"/assets/img/icon-arrow-right-white.svg" | relative_url }} width="15" height="15"/>
+                <img src={{"/assets/img/icon-arrow-right-white.svg" | relative_url }} alt="" width="15" height="15"/>
```

Comment:
Out of curiosity, why are some of these `alt` attributes empty?  I'm assuming it's intentional.

Reply:
That makes a lot of sense.  Thanks for the comments - I learned something today